features.txt

```diff
@@ -5,6 +5,10 @@
 # dedicated feature flag so that consumers may more easily omit them as
 # necessary.
 #
+# Generally, feature flags should be per proposal, not per individual API
+# that a proposal adds. To filter for specific APIs, use the directory
+# structure.
```

Comment:
this is unfortunate advice imo because test262harness makes it much easier to exclude feature flags than to target "everything, minus specific directories", but there may be alternative solutions there.

Reply:
This is nearly the same discussion as on https://github.com/tc39/test262/pull/3760 — I'd be in favour of adding exclude paths to test262-harness.